diag_manager/fms_diag_axis_object.F90

```diff
@@ -107,13 +108,13 @@ module fms_diag_axis_object_mod
   !> @brief Type to hold the subaxis
   !> @ingroup diag_axis_object_mod
   TYPE, extends(fmsDiagAxis_type) :: fmsDiagSubAxis_type
-    CHARACTER(len=:), ALLOCATABLE, private  :: subaxis_name   !< Name of the subaxis
-    INTEGER                      , private  :: starting_index !< Starting index of the subaxis relative to the
-                                                              !! parent axis
-    INTEGER                      , private  :: ending_index   !< Ending index of the subaxis relative to the
-                                                              !! parent axis
-    type(subRegion_type)         , private  :: subRegion      !< Bounds of the subaxis (lat/lon or indices)
-    INTEGER                      , private  :: parent_axis_id !< Id of the parent_axis
+    CHARACTER(len=:),  ALLOCATABLE , private  :: subaxis_name   !< Name of the subaxis
```

Comment:
There's some weird spacing here

Reply:
i think it looks nice
https://github.com/NOAA-GFDL/FMS/blob/cd34f1f1d0dd185e2864c39a88fbd903667c539c/diag_manager/fms_diag_axis_object.F90#L110-L120